### ISSUE TITLE
Users should login to their default stores as set in mSupply Cloud?

### ISSUE THREAD

**Issue by nisha-dangol** [openmsupply/open-msupply#2099]
## What went wrong? 😲
If there are multiple stores within a site with multiple users, I think the users should be able to login to their default store after signing in with their credentials. (since oms doesn't have option to choose store to login unlike the mSupply desktop with stores DDL to choose to login)

## Expected behaviour 🤔
Users should login to their default store as set in mSupply Cloud!

## How to Reproduce 🔨
Steps to reproduce the behaviour:
1. Have multiple stores within a site with multiple users, for eg: have `user1`'s default store as `store1` and `user2` on `store2`
2. Go to login page for oms
3. Throw in the credentials for `user1`
4. You'll (by default) login to `store1` 
5. Log out
6. Now throw in credentials for 'user2'
7. See error: you'll still login to 'store1' (even when default store is set as 'store2`)

## Screenshots/Recording
https://recordit.co/Bd8iQ1P7V3

## Your environment 🌱

 - Version:  v1.3.00-test
 - Platform:
   - [ ] android (tablet)
   - [x] browser (extra points if you tell us which one)
   - [x] desktop (windows)
   - [ ] desktop (macOS)
   - [x] server (windows)

**Comment by nisha-dangol** [comment]
## Testing
Tested in v1.3.00 exe

### Setup
- [x] You'll need at least two users, with one of them having multiple stores available.

### Tests
- [x] Clear local storage: localStorage.removeItem('@openmsupply-client/mru/credentials')
- [x] Login as user_a, see that they are assigned their default store
- [x] logout, login as user_b, see that they are assigned their default store
- [x] change the store (store_b), logout and login. You should be in store_b
- [x] login as user_a: they'll be in user_a::default_store

### Basically,
- [x] the first time a user logs in, the store should be their default. - PASS
- [x] If a user changes stores, then logs out and in again, the store should be the most recently used store. - PASS
